sonar-project.properties

```diff
@@ -14,4 +14,4 @@ sonar.python.version=3.8, 3.9
 sonar.plsql.file.suffixes=""
 
 # exclude non-API worker files from coverage report
-sonar.coverage.exclusions=database/**,website/**,workers/scripts/**,workers/tests/**
\ No newline at end of file
+sonar.coverage.exclusions=database/**,website/**,workers/scripts/**,workers/tests/**
```

Comment:
this change isn't necessary and could be reverted.

Reply:
I reverted this change using an interactive rebase so it will not show up as changed file anymore.